src/mumble/PipeWire.cpp

```diff
@@ -0,0 +1,437 @@
+// Copyright 2021 The Mumble Developers. All rights reserved.
+// Use of this source code is governed by a BSD-style license
+// that can be found in the LICENSE file at the root of the
+// Mumble source tree or at <https://www.mumble.info/LICENSE>.
+
+#include "PipeWire.h"
+
+#include "Global.h"
+
+#include <pipewire/core.h>
+#include <pipewire/keys.h>
+#include <pipewire/stream.h>
+
+#include <spa/param/audio/format-utils.h>
+#include <spa/pod/builder.h>
+
+#ifdef Q_CC_GNU
+#	define RESOLVE(var)                                                    \
+		{                                                                   \
+			var = reinterpret_cast< __typeof__(var) >(m_lib.resolve(#var)); \
```

Comment:
```suggestion
			var = reinterpret_cast< decltype(var) >(m_lib.resolve(#var)); \
```
-> That makes it standard-conform and not compiler-depdenent